static/src/javascripts/projects/commercial/modules/hosted/gallery.spec.js

```diff
@@ -0,0 +1,158 @@
+// @flow
+import interactionTracking from 'common/modules/analytics/interaction-tracking';
+import { noop } from 'lib/noop';
+import HostedGallery from './gallery';
+import { galleryHtml } from './gallery-html';
+
+jest.mock('lib/detect', () => ({
+    hasPushStateSupport: jest.fn(),
+    getBreakpoint: jest.fn(),
+    hasTouchScreen: jest.fn(),
+}));
+jest.mock('common/modules/analytics/interaction-tracking', () => ({
+    trackNonClickInteraction: jest.fn(() => Promise.resolve()),
+}));
+jest.mock('lib/load-css-promise', () => ({
+    loadCssPromise: Promise.resolve(),
+}));
+jest.mock('commercial/modules/dfp/performance-logging', () => ({
+    moduleStart: jest.fn(),
+    moduleEnd: jest.fn(),
+}));
+
+let gallery: any;
+
+describe('Hosted Gallery', () => {
+    beforeEach(done => {
+        if (document.body) {
+            document.body.innerHTML = galleryHtml;
+        }
+        HostedGallery.init().then(galleryInstance => {
+            gallery = galleryInstance;
+            done();
+        });
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+
+        if (document.body) {
+            document.body.innerHTML = '';
+        }
+    });
+
+    it('should exist', done => {
+        expect(gallery).toBeDefined();
+        done();
+    });
+
+    const classListFor = (className: string): string =>
+        // $FlowFixMe
+        document.querySelector(`.${className}`).classList.toString();
```

Comment:
Are these `$FlowFixMe`s because `document.querySelector()` might return `null`? In which case, is it worth storing the result of the `querySelector` call to a variable, null checking and returning an empty string as a fallback? Or is this approach a bit clunky?